.github/workflows/universal_decoder_ci.yaml

```diff
@@ -13,6 +13,12 @@ on:
     paths:
       - 'Samples/UniversalDecoder/**'
   workflow_dispatch:
+    inputs:
+      pushDockerImage:
```

Comment:
Since the pipeline is not enforced to succeed on every PR run, I would argue that instead of adding complexity by introducing new parameters, we should instead consider just removing the triggers that run the pipeline on each PR (and maybe only run it after a merge). What do you think?